### ISSUE TITLE
Cannot connect weaviate-cli, but weaviate config is OK.

### ISSUE THREAD

**Issue by glodov** [weaviate/weaviate#3540]
URL and secret are changed because of the privacy.
```bash
weaviate config view                                                       
{
    "url": "https://yaro-laws-b61fnja1.weaviate.network",
    "auth": {
        "type": "client_secret",
        "secret": "GJLPaLbzKBNqK9NISDFVmmNOveaFe5uhfMnS"
    }
}
```
Error I have:
```bash
weaviate ping                                                              
Traceback (most recent call last):
  File "/Users/yaro/miniconda3/bin/weaviate", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/semi/misc.py", line 21, in main_ping
    ping(get_client_from_context(ctx))
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/semi/utils.py", line 16, in get_client_from_context
    return ctx.obj["config"].get_client()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/semi/config/commands.py", line 103, in get_client
    return weaviate.Client(self.config["url"], cred)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/weaviate/client.py", line 150, in __init__
    self._connection = Connection(
                       ^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/weaviate/connect/connection.py", line 165, in __init__
    self._create_session(auth_client_secret)
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/weaviate/connect/connection.py", line 222, in _create_session
    self._session = _auth.get_auth_session()
                    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/weaviate/connect/authentication.py", line 76, in get_auth_session
    session = self._get_session_client_credential(self._credentials)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yaro/miniconda3/lib/python3.11/site-packages/weaviate/connect/authentication.py", line 127, in _get_session_client_credential
    raise MissingScopeException
weaviate.exceptions.MissingScopeException
```
Version:
```bash
weaviate version
2.2.0
```

**Comment by dudanogueira** [comment]
Hi! This is not an issue, but more [likely a forum thread](https://forum.weaviate.io/)
Make sure to use our latest python client: 3.24.1 as I write:
https://github.com/weaviate/weaviate-python-client/

Also, check here for detailed instructions on how to connect to your  cluster:
https://weaviate.io/developers/weaviate/quickstart#step-3-connect-to-weaviate

I will close this issue for now, but feel free to reopen or open a new forum thread for it.

Thanks!

**Comment by glodov** [comment]
Thank you! 🙏
I am new to Python on such a scale.
My Python scripts are working well, with no issues.
So it means the version is OK.

But when I ran no package was found.
```bash
$ pip3 show weaviate-python-client
WARNING: Package(s) not found: weaviate-python-client
```

Script `version.py` did not help as well
```python
import weaviate
print("Weaviate Python Library Version:", weaviate.__version__)
```

```bash
$ python version.py 
Weaviate Python Library Version: unknown version
```

I have also set up a virtual env for weaviate in Python to install the latest version of the client library, because I could not do it in a global environment, because of dependency conflicts.
But it did not help.
The files are updated, but the same issue, not on 127, but on 130 line is thrown.

But, I just wanted to clean up a database with weaviate-cli tool.  
The best solution for now is not to use it for me. 😄
